vertical-pod-autoscaler/pkg/recommender/Dockerfile

```diff
@@ -31,5 +31,4 @@ ARG TARGETARCH
 
 COPY --from=builder /gopath/src/k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/recommender-$TARGETARCH /recommender
 
-ENTRYPOINT ["/recommender"]
-CMD ["--v=4", "--stderrthreshold=info", "--prometheus-address=http://prometheus.monitoring.svc"]
```

Comment:
Should this be moved into the recommender as a default value?

Reply:
I accidentally deleted the ENTRYPOINT as well ( will fix ) , the prometheus address will be configured as default here: https://github.com/kubernetes/autoscaler/pull/7312/files#diff-cf801d600c83feda2a9ee187a09a43380478c36df274f42f2246f2de2a203505R48
You are correct, I should changed its name to  `prometheus-address`

Reply:
That seems wrong. `address` and `prometheus-address` are two different parameters

Reply:
`prometheus-address` is listed here: https://github.com/kubernetes/autoscaler/pull/7312/files#diff-bc201689c0c988297d4f93cf844937aa4f4755bfdc5bc912746148eaeed3bb64R61
And it has no default

Reply:
Oh yes I see sorry ( got confused ) . I guess I can move set it too as a default value.